README.md

```diff
@@ -67,30 +67,79 @@ import { SoundPlugin, Container } from 'springroll-container';
   });
 	container.openPath('game.html');
 ```
-UISizePlugin, LayersPlugin, DifficultyPlugin:
+DifficultyPlugin:
+```javascript
+import { DifficultyPlugin, Container } from 'springroll-container';
+
+  const container = new springroll.Container({
+    iframeSelector: "#game",
+    plugins: [
+      //The DifficultyPlugin offers many different types of "difficulty" so your game can offer fine grained control over their experience to the user. Explanations of the options are below
+      //DifficultyPlugin also accepts an [optional] initial value for its difficulty types
+      new DifficultyPlugin({
+        //Sliders expect an HTML Input Element of type="range"
+        hitAreaScaleSlider: '#hitAreaScaleSlider',
+        //All difficulty values run 0.0 to 1.0. Default = 0.5
+        defaultHitAreaScale = 0.5,
+        dragThresholdScaleSlider: '#dragThresholdScaleSlider',
+        defaultDragThresholdScale = 0.5,
+        healthSlider: '#healthSlider',
+        defaultHealth = 0.5,
+        objectCountSlider: '#objectCountSlider',
+        defaultObjectCount = 0.5,
+        completionPercentageSlider: '#completionPercentageSlider',
+        defaultCompletionPercentage = 0.5,
+        speedScaleSlider: '#speedScaleSlider',
+        defaultSpeedScale = 0.5,
+        timersScaleSlider: '#timersScaleSlider',
+        defaultTimersScale = 0.5,
+        inputCountSlider: '#inputCountSlider',
+        defaultInputCount = 0.5,
+      }),
+    ]
+  });
+	container.openPath('game.html');
+```
+DifficultyPlugin Options:
+
+See the [Springroll Application Docs](https://github.com/SpringRoll/SpringRoll/tree/develop/src#difficulty-features "Springroll Application Documentation") for more detailed information.
```

Comment:
I'm leery about pointing to the docs here because I don't want the link to break all of a sudden. Moreover, we're pointing to `develop` which might be less "stable". I think for the time being, let's just remove the link (even though I _do_ want to have it there). I think eventually we'll just need to point to springroll.io or something.